src/data/faq.json

```diff
@@ -566,11 +566,14 @@
                         "anchor": "encounter_criteria",
                         "active": true,
                         "textblock": [
-                            "The Corona Warn App has three different status indicators: \"unknown risk\" (grey), \"low risk\" (green) and \"increased risk\" (red).",
+                            "The calculation of your individual risk is based on the following factors:",
+                            "<ol><li> duration of the encounter </li><li> Distance at the time of the encounter </li><li> Time that has elapsed since the encounter </li><li>Infectiousness (derived from information on when and whether symptoms occur).</li></ol>",
```

Comment:
Capitalise duration, so
duration of the encounter => Duration of the encounter

Reply:
thanks